README.md

```diff
@@ -14,3 +14,4 @@ This repository contains samples for creating an Azure Marketplace Kubernetes of
 ## Samples in this repo
 
 - azure-vote: The Azure Vote application is a simple sample used throughout the Azure Container Service (AKS) documentation. The application consists of two pods, one running a flask web form, and the second a redis instance for temporary data storage.
+- azure-vote-custom: The Azure Vote Custom application is a simple sample to demonstrate custom meters usage in a containerized app. The application consists of two pods, one running a flask web form, and the second a redis instance for temporary data storage. This application can be used as a sample app for custom meters based offers on Azure marketplace
```

Comment:
Can you please update the description to
1. For k8s-offer-azure-vote -> simple sample used throughout the Kubernetes Applications Marketplace documentation.
2. k8s-offer-azure-vote-custom-meters -> The Azure Vote custom meters application is a simple sample to demonstrate custom meters usage in a Kubernetes application. The application consists of two pods, one running a flask web form, and the second a redis instance for temporary data storage. The application submits usage data to Azure Commerce Metering API for every vote casted per hour.